src/go/rpk/pkg/config/params.go

```diff
@@ -646,6 +651,15 @@ func (p *Params) InstallCloudFlags(cmd *cobra.Command) {
 	cmd.MarkFlagsRequiredTogether(FlagClientID, FlagClientSecret)
 }
 
+func (p *Params) InstallFormatFlag(cmd *cobra.Command) {
+	pf := cmd.PersistentFlags()
```

Comment:
Why **can't** we? I used persistent here because I only wanted to install the flag in one command (top level acl user).

The alternative is installing format flag in every command that we need it.

Reply:
Technically we can, but this _might_ add the flag to commands that don't yet support --format, which may be confusing. We have some prior (tedious) existing pattern of having enabled/disabled in every command -- I expect we'd add `--format` only to commands that support it.

In this PR I _think_ it's fine, so maybe it's ok to keep for now.